.github/workflows/rubyandnode.yaml

```diff
@@ -0,0 +1,23 @@
+name: Ruby and Node
+
+on:
```

Comment:
I _think_ we probably want to use `pull_request` here...

```
on:
  pull_request:
    types: [ opened, synchronize, reopened ]
```

this should:

* avoid it trying to run the Action for every single commit in the PR (just do the HEAD)
* possibly avoid running after merging `master` (no point, it's too late - not 100% sure if `push` counts here, it's not clear)

but LGTM